app_HSV_similar_product.py

```diff
@@ -1,19 +1,28 @@
+# -*- coding: utf-8 -*-
 import dash
 import dash_core_components as dcc
 import dash_html_components as html
-import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
 from dash.dependencies import Input, Output, State
+import numpy as np
+import pandas as pd
+from math import sqrt
+from colorutils import Color 
 
 # launch app with customer css
 app = dash.Dash(__name__)
 app.title = 'Foundation Shades Across the Globe'
-shades = pd.read_csv("./data/shades_processed.csv") # read processed data
-df = px.data.tips() # delete this next week
+shades = pd.read_csv("./data/shades_processed.csv", encoding="utf-8-sig") # read processed data
+shades['hex'] = '#' + shades['hex'] # add # symbol for displaying color
```

Comment:
Should we preprocess the data so this is the case and we don't need to process the data in app.py?

Reply:
Hex numbers are 16-bit numbers. I guess we can if there are no other places that will need the original Hex for conversion. I will do it after the next PR.